### DISCUSSION TITLE
Нужен ли USB накопитель после всех процедур?

### DISCUSSION THREAD

**Discussion by Noksa** [Anonym-tsk/nfqws-keenetic#16] [Q&A] [answered]
Что-то нигде не вижу этой информации.
После установки и настройки - можно ли usb накопитель вытыкать из роутера и благополучно забыть про него? Или он должен вечно торчать в роутере?

**Comment by IronButterfly** [comment, marked as answer]
Конечно должен быть воткнут. У вас же на нем все окружение Entware развернуто, если вы его для этого выбирали, а не внутреннюю память роутера.

**Comment by Noksa** [comment]
@IronButterfly Это печально, ведь этот слот на даче у меня используется под модем. Хаб что ли брать :(

**Reply by IronButterfly** [reply]
Можно же Entware на внутреннюю память роутера развернуть, хотя и не рекомендуется. Если она навернётся из-за интенсивного использования софта Entware, то всему роутеру конец. Но это все в теории, на практике много кто так использует и вроде проблем не имеют.
